ldk/javascript/src/types/oliveHelps/whisper.d.ts

```diff
@@ -57,6 +57,8 @@ declare namespace WhisperService {
 
   type WidthSize = 'full' | 'half';
 
+  type CustomHeight = '8rem' | '16rem' | '24rem' | '36rem';
```

Comment:
I believe current pattern is that we are sending text definition of sizes to the Sidekick and converting to rems in the react component it self.
Take a look as the example:
`export enum StyleSize {
  None = 'none',
  Small = 'small',
  Medium = 'medium',
  Large = 'large',
}
`